docs/user_guide/datasets/acs_income.rst

```diff
@@ -0,0 +1,165 @@
+   *  - Column name
+      - Description
+
+   *  - AGEP
+      - Age as an integer from 0 to 99
+
+   *  - COW
+      - Class of worker:
+         1. Employee of a private for-profit company or business, or of an individual, for wages, salary, or commissions 
+         2. Employee of a private not-for-profit, tax-exempt, or charitable organization 
+         3. Local government employee (city, county, etc.) 
+         4. State government employee 
+         5. Federal government employee 
+         6. Self-employed in own not incorporated business, professional practice, or farm 
+         7. Self-employed in own incorporated business, professional practice or farm 
+         8. Working without pay in family business or farm 
+         9. Unemployed and last worked 5 years ago or earlier or never worked
+
+   *  - SCHL
+      - Educational attainment:
+         1. No schooling completed
+         2. Nursery school or preschool
+         3. Kindergarten
+         4. Grade 1
+         5. Grade 2
+         6. Grade 3
+         7. Grade 4
+         8. Grade 5
+         9. Grade 6
+         10. Grade 7
+         11. Grade 8
+         12. Grade 9
+         13. Grade 10
+         14. Grade 11
+         15. Grade 12 (no diploma)
+         16. Regular high school diploma
+         17. GED or alternative credential
+         18. Some college but less than 1 year
+         19. 1 or more years of college credit but no degree
+         20. Associate's degree
+         21. Bachelor's degree
+         22. Master's degree
+         23. Professional degree beyond a bachelor's degree
+         24. Doctorate degree
+
+   *  - MAR
+      - Marital status:
+         1. Married
+         2. Widowed
+         3. Divorced
+         4. Separated
+         5. Never married or under 15 years old
+
+   *  - OCCP
+      - Occupation. There are over 500 categories. Please see data dictionary at `ACS PUMS documentation <https://www.census.gov/programs-surveys/acs/microdata/documentation.2018.html>`_ for the full list of occupation codes.
+
+   *  - POBP
+      - Place of birth. There are over 200 categories, including the 50 US states and several countries. Please see the data dictionary at `ACS PUMS documentation <https://www.census.gov/programs-surveys/acs/microdata/documentation.2018.html>`_ for the full list.
+
+   *  - RELP
+      - Relationship:
+         0. Reference person
+         1. Husband or wife
+         2. Biological son or daughter
+         3. Adopted son or daughter
+         4. Stepson or stepdaughter
+         5. Brother or sister
+         6. Father or mother
+         7. Grandchild
+         8. Parent-in-law
+         9. Son-in-law or daughter-in-law
+         10. Other relative
+         11. Roomer or boarder
+         12. Housemate or roommate
+         13. Unmarried partner
+         14. Foster child
+         15. Other nonrelative
+         16. Institutionalized group quarters population
+         17. Noninstitutionalized group quarters population
```

Comment:
What do these actually mean? Any idea? I'm not familiar with these terms

Reply:
According to [disabilitystatistics](https://www.disabilitystatistics.org/glossary.cfm?g_id=266&view=true#nextTerm263), "institutionalized population includes people under formally authorized supervised care," and those people are generally referred to as "inmates" or "patients" and have restricted access to the surrounding community, whereas "the noninstitutionalized population includes all people who live in group quarters other than institutions, such as dormitories, halfway houses, and communes."

Reply:
I believe the Census uses these terms to distinguish group quarters like university dorms from group quarters like prison facilities. Though I agree that a footnote explaining the distinction would be helpful! Here's more background: https://www.census.gov/topics/income-poverty/poverty/guidance/group-quarters.html

Reply:
Couldn't agree more with both of you here! I find this context very useful to understand as well.